<ui version="4.0" >
 <class>StandardWidgetClass</class>
 <widget class="QWidget" name="StandardWidgetClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>StandardWidget</string>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>90</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>SimSun</family>
     <pointsize>30</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>标准人查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>SimSun</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>请输入编号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberLineEdit" >
   <property name="geometry" >
    <rect>
     <x>430</x>
     <y>60</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>SimSun</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>SimSun</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>请选择模型：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="modelComboBox" >
   <property name="geometry" >
    <rect>
     <x>430</x>
     <y>130</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView" >
   <property name="geometry" >
    <rect>
     <x>680</x>
     <y>30</y>
     <width>171</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchBtn" >
   <property name="geometry" >
    <rect>
     <x>370</x>
     <y>180</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>SimSun</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>查  找</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>250</y>
     <width>271</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
